pandas/core/indexing.py

```diff
@@ -737,17 +737,6 @@ def _maybe_mask_setitem_value(self, indexer, value):
 
         return indexer, value
 
-    @final
-    def _tupleize_axis_indexer(self, key) -> tuple:
```

Comment:
I agree with this change (if possible try not to have code changes in typing PRs) but now it is possible that sub-classes can implement `_tupleize_axis_indexer` (previously this would be prevented by mypy+final) - obviously a very minor/theoretical "concern".

cc @jbrockmendel if there was a reason to have this function as a method.

Reply:
> if there was a reason to have this function as a method.

it uses self.axis and self.ndim, so either way

Reply:
@twoertwein that makes sense. Are minor changes like this okay, or is it better if I submit those as a separate refactoring first?

I don't think overriding `_tupleize_axis_indexer` is possible anymore because it's not a class method. Am I overlooking something?

Reply:
> Are minor changes like this okay, or is it better if I submit those as a separate refactoring first?

I would definitely keep the variable name changes as part of the typing PRs. I think the only risk of combining typing+code changes is that the code review might take longer (not everyone is comfortable with the code (me) and others might not be comfortable reviewing typing changes).

Reply:
> it uses self.axis and self.ndim, so either way

ups, I think I missed that. What errors would you get if you keep it as a method?

Reply:
it wouldn't cause any errors. The important change here is that it receives `axis` as an argument, so that mypy can assert that it will not be `None` (since it is called inside `if self.axis is not None:`).